components/src/core/components/CardTable/Table/table-header-sort-dropdown.scss

```diff
@@ -40,11 +40,14 @@
         margin-left: 5px;
       }
       &:hover,
-      &:focus-visible,
       &:focus,
+      &:focus-visible,
       &.--selected {
         background-color: $oxd-dropdown-item-hover-color;
       }
+      &:not(:first-child) {
+        margin-top: 5px;
```

Comment:
as discussed this is suggested by mike, after reviewing. This needed to be added to the defect when marking it as fixed

Reply:
This is added by Mike
https://orangehrmenterprise.atlassian.net/browse/RMP-454?focusedCommentId=15154